fallback_data.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"os"
+)
+
+func getFallbackData() (*imageData, error) {
+	if len(conf.FallbackData) > 0 {
+		return base64FallbackData(conf.FallbackData)
+	}
+
+	if len(conf.FallbackPath) > 0 {
+		return fileFallbackData(conf.FallbackPath)
+	}
+
+	if len(conf.FallbackURL) > 0 {
+		return remoteFallbackData(conf.FallbackURL)
+	}
+
+	return nil, nil
+}
+
+func base64FallbackData(encoded string) (*imageData, error) {
+	data, err := base64.StdEncoding.DecodeString(encoded)
+	if err != nil {
+		return nil, fmt.Errorf("Can't decode fallback data: %s", err)
+	}
+
+	imgtype, err := checkTypeAndDimensions(bytes.NewReader(data))
+	if err != nil {
+		return nil, fmt.Errorf("Can't decode fallback: %s", err)
+	}
+
+	return &imageData{Data: data, Type: imgtype}, nil
+}
```

Comment:
Let's rename `base64WatermarkData` to `base64ImageData` and reuse it instead of duplicating the code. Same for the further functions. All the `imageData` loading functions and the type itself can be put to `image_data.go`.